postgres/tests/compose/docker-compose-latest-alpine.yaml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  postgres:
+    image: "postgres:alpine"
```

Comment:
can you use a new `POSTGRES_IMAGE` environment variable in the current docker-compose, so that we keep one docker-compose file only